papermill/cli.py

```diff
@@ -171,10 +217,22 @@ def papermill(
         parameters_final.update(read_yaml_file(files))
     for params in parameters_yaml or []:
         parameters_final.update(yaml.load(params, Loader=NoDatesSafeLoader))
-    for name, value in parameters or []:
-        parameters_final[name] = _resolve_type(value)
-    for name, value in parameters_raw or []:
-        parameters_final[name] = value
+    for name_values in parameters or []:
+        name, values = name_values.pop(0), name_values
```

Comment:
I _think_ this is causing the test failures.
```suggestion
        name, values = name_values[0], name_values[1:]
```

Reply:
Hmmm, interesting that they may be the cause.

`name_values[1:]` could raise `IndexError`...

Reply:
Nope it's safe:

```
In [1]: t = []                                                                                                                              
In [2]: t[1:]                                                                                                                               
Out[2]: []
```

Reply:
Ah.. thanks.

Reply:
Ah, I get it now. `name_values` is a `tuple`, not a `list`.